third_party/babel/custom-babel-helpers.js

```diff
@@ -0,0 +1,88 @@
+(function (global) {
```

Comment:
Stupid question: Did you make any changes to this file? Could we just generate it during build?